### DISCUSSION TITLE
Can't find newly installed apps

### DISCUSSION THREAD

**Discussion by jbjoeB** [i3/i3#4711] [Q&A]
So i installed gparted and discord

Discord from the debian repositoires and discord via snap.
I can sudo from the terminal or run discord from the terminal but not from dmenu, i've rebooted also and dosen't work
Any ideas?

**Comment by CreadDotson** [comment]
dmenu is going to check /usr/bin and /bin. It's possible that those applications installed somewhere else in your path. I would use "whereis discord" or whatever command you want to add and then add a link to that result in the /usr/bin directory. There might be a better way to do this. If someone knows that please reply. Add a link with the ln command btw
